src/main/java/pinger/Pinger.java

```diff
@@ -44,6 +44,6 @@ private static String readAllLines(InputStream stream) throws IOException {
     }
 
     private static void addField(String name, Object value) {
-        System.out.printf(name + ": %s", value);
+        System.out.println(name + ": %s", value);
```

Comment:
This does not compile. `name + ": %s", value` needs to be changed to `name + ": " + value`.